app/controllers/provider/admin/messages/inbox_controller.rb

```diff
@@ -22,11 +22,16 @@ def reply
     reply = @message.reply
     reply.attributes = message_params
 
-    reply.save!
-    reply.deliver!
-
-    flash[:notice] = 'Reply was sent.'
-    redirect_to action: :index
+    if reply.valid?
```

Comment:
This prevents sending messages without a subject, but by the title I understood that messages without a body should fail as well. Am I right?

Reply:
This is subtask, and we have already added validation on message subject which check if message is valid or not . same goes for reply also, as reply is also a message and as per discussion with team its only required to make subject as mandatory field and not body